docker/api/service.py

```diff
@@ -0,0 +1,62 @@
+from ..utils import check_resource
+
+
+class ServiceApiMixin(object):
+    def services(self, name=None, partial_id=None):
+        url = self._url("/services")
+        params = {
+            'name': name,
+            'partial-id': partial_id,
+        }
+        res = self._get(url, params=params)
+        return self._result(res, json=True) or []
+
+    def publish_service(self, name, network_name=None):
+        data = {
+            'name': name,
+            'network_name': network_name,
+        }
+        url = self._url("/services")
+        res = self._post_json(url, data=data)
+        return self._result(res, json=True)
+
+    def unpublish_service(self, service_id):
+        url = self._url("/services/{0}".format(service_id))
+        res = self._delete(url)
+        self._raise_for_status(res)
+
+    def inspect_service(self, service_id):
+        url = self._url("/services/{0}".format(service_id))
+        res = self._get(url)
+        return self._result(res, json=True)
+
+    def service_backends(self, service_id):
+        url = self._url("/services/{0}/backend".format(service_id))
+        res = self._get(url)
+        return self._result(res, json=True)
+
+    def sandboxes(self, partial_container_id=None):
+        url = self._url("/sandboxes")
+        params = {}
+        if partial_container_id:
```

Comment:
`params = { 'partial-container-id': partial_container_id }` would do the same thing here in practice.